integrations/clickup/.port/resources/blueprints.json

```diff
@@ -0,0 +1,197 @@
+[
+    {
+        "identifier": "clickupTeam",
+        "description": "This blueprint represents a Clickup team",
+        "title": "Clickup Team",
+        "icon": "Clickup",
+        "schema": {
+            "properties": {
+                "url": {
+                    "type": "string",
+                    "title": "URL",
+                    "format": "url"
+                },
+                "members": {
+                    "type": "array",
+                    "title": "Members",
+                    "items": {
+                        "type": "string",
+                        "format": "user"
+                    }
+                }
```

Comment:
is this property schema and the one in assignees in clickup issue should be the same ? ( as they include list of users ) 
```
                "assignees": {
                    "title": "Assignees",
                    "type": "array",
                    "items": {
                        "type": "string",
                        "format": "email",
                        "description": "The users assigned to the issue"
                    }
                },
```
if so there is inconsistency please elaborate on the reason

Reply:
Updated to use user format